ouroboros-consensus/src/Ouroboros/Consensus/Node/Tracers.hs

```diff
@@ -48,7 +48,7 @@ data Tracers' peer blk f = Tracers
   , blockFetchClientTracer        :: f (TraceLabelPeer peer (TraceFetchClientState (Header blk)))
   , blockFetchServerTracer        :: f (TraceBlockFetchServerEvent blk)
   , txInboundTracer               :: f (TraceTxSubmissionInbound  (GenTxId blk) (GenTx blk))
```

Comment:
Would it make sense to add it to `txInboundTracer` too?

Reply:
Yes.

Reply:
I'll add a commit which does that.